client/src/components/desktop/ResourceDetail.jsx

```diff
@@ -390,6 +394,17 @@ function ResourceDetail(props) {
               id: `resource-eligibilityRequirements-${match.params.id}`,
               defaultMessage: eligibility,
             })}`}
+          {lastUpdated && (
+            <t style={{ color: 'gray' }}>
```

Comment:
what is a t tag? is there a reason we can't do span/div?

Reply:
bruh idk I swear it was used for text but that might've been a completely different project lol